templates/home/index.html.twig

```diff
@@ -40,37 +40,25 @@
     </div>
     <ul class="list-group m-4">
         <li class="list-group-item list-group-item-info">Projektai</li>
-        {% if projectArray %}
-            {% for key in projectArray|keys %}
+        {% if studentArray %}
+            {% for key, project in studentArray %}
                 <li class="list-group-item">
-                    <h3>{{ projectArray[key]['name'] }}</h3>
+                    <h3>{{ project.name }}</h3>
                     <div class="panel panel-default">
                         <div>
-                            <a target="_blank" href="{{ projectArray[key]['github'] }}">
-                                {{ projectArray[key]['githubDecode']|replace
-                                    ({
-                                        'https://': '',
-                                        'http://': '',
-                                        'www.': ''
-                                    })
-                                }}
+                            <a target="_blank" href="{{ 'https://github.com/nfqakademija/' ~ key|url_encode }}">
+                                {{ 'github.com/nfqakademija/' ~ key }}
```

Comment:
:+1: Įdomus sprendimas

Šiaip būtų užtekę ir:
```twig
<a target="_blank" href="https://github.com/nfqakademija/{{ key|url_encode }}">
                                github.com/nfqakademija/{{ key }}
```
Nes pirmas argumentas niekada nesikeičia ir jo vis tiek nei įvykdomas, nei veikia `url_encode`

Plačiau:
 * https://stackoverflow.com/questions/7704253/how-to-concatenate-strings-in-twig